### DISCUSSION TITLE
Missing avatar images

### DISCUSSION THREAD

**Discussion by lockered** [amborle/featmap#39] [General]
Hi,
yesterday I have noticed that on my local instance of featmap all avatar images are missing. I think this happened because I have forced npm to fix all the security issues. 
I looks to me, as if the actual way of defining the images using "require" is not working anymore. Is there a known solution? 
I have found some hints on Stack Overflow, but have not tried to follow these hints until now.

**Comment by amborle** [comment]
I am not surprised that the app breaks if you update npm. Will investigate when there is time. Thanks for the hint on "require".

**Comment by lockered** [comment]
Because with the update of npm the number of severe vulnerabilities goes up, I have naivly tried the automatic fixing and it took some days until  I found the avatar issue.  Nothing else found so far.

By the way: I am astonished that the whole npm thing works...

**Comment by amborle** [comment]
Hi, updated some libs with vulnerabilities and fixed also the avatar issue that pops up.
